docs/MQTT-streamingsource.md

```diff
@@ -0,0 +1,116 @@
+MQTT Streaming Source
+=====================
+
+[![Join CDAP community](https://cdap-users.herokuapp.com/badge.svg?t=mqtt)](https://cdap-users.herokuapp.com)
```

Comment:
This goes in plugin documentation as reference and it should not have all these links.